independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -310,12 +310,14 @@ private ClassPathResourceIndex getClassPathResourceIndex() {
 
                     for (ClassPathElement bannedElement : bannedElements) {
                         classPathResourceIndexBuilder.scanClassPathElement(bannedElement,
-                                classPathResourceIndexBuilder::addBannedResource);
+                                (classPathElement, resource) -> classPathResourceIndexBuilder.addBannedResource(
+                                        resource));
```

Comment:
Yeah so the reason why I had two parameters was to have a consistent interface for anything called with `scanClassPathElement`.

Do you think dropping the parameter is worth adding a real lambda?

Reply:
My main gripe about this is that the code looked simpler and more consistent the way I did it but I suppose it's in the eyes of the reader.

Reply:
I personally I have never seen a method parameter being added just so a method reference can be used. I'm not saying  it's bad, it's just very unintuitive.

Reply:
Well, if you think in terms of contract of the `scanClassPathElement`, I think it makes kinda sense.

Anyway, probably not worth discussing it.